client/components/user/ProfileContainer.js

```diff
@@ -84,13 +80,18 @@ export default function ProfileContainer() {
       {isLoading ? (
         <Text>Loading....</Text>
       ) : (
-        <Box
-          style={{ width: "100%", height: "100%" }}
-          space={4}
-          alignItems="center"
-        >
-          <UserDataContainer data={packsData} type="packs" />
-          <UserDataContainer data={tripsData?.trips} type="trips" />
+        <Box style={styles.mainContentContainer}>
+          <Box style={styles.userDataContainer}>
+            <UserDataContainer data={favoritesData} type="favorites" />
```

Comment:
What do we show if user does not have any favorites? Current button does not link to anything.